src/django_bootstrap5/html.py

```diff
@@ -1,9 +1,13 @@
+from copy import copy
+
 from django.forms.utils import flatatt
 from django.utils.html import format_html
 
 from django_bootstrap5.text import text_value
 from django_bootstrap5.utils import get_url_attrs
 
+from .core import get_bootstrap_setting
```

Comment:
Let’s keep using absolute imports?
```suggestion
from django_bootstrap5.core import get_bootstrap_setting
```